### DISCUSSION TITLE
Is it possible to hide the right side bar (i.e., "On this page")?

### DISCUSSION THREAD

**Discussion by codylindley** [withastro/starlight#1712] [Q&A] [answered]
### What version of `starlight` are you using?

0.21.3

### What is your idea?

Hide the right side bar (i.e., "On this page"! 

Is there some reason I can't hide this bar?

### Why is this feature necessary?

For pages in the TOC that have only one heading.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by liruifengv** [comment, marked as answer]
Have you tried the frontmatter property?

https://starlight.astro.build/reference/frontmatter/#tableofcontents

**Comment by codylindley** [comment]
Ah, yes. That's it. Thank you. Not exactly obvious as I thought the left-hand column contained the TOC.
